integration/v3_lease_test.go

```diff
@@ -247,6 +247,30 @@ func TestV3PutOnNonExistLease(t *testing.T) {
 	}
 }
 
+// TestV3GetNonExistLease tests a case where using LeaseTimeToLive to retrive a nonexistent lease should return a lease not found error.
+// A node panic bug is found when a non leader etcd server returns nil error instead of lease not found error. That causes the server to crash.
+// related issue https://github.com/coreos/etcd/issues/6537
+func TestV3GetNonExistLease(t *testing.T) {
+	defer testutil.AfterTest(t)
+	clus := NewClusterV3(t, &ClusterConfig{Size: 3})
+	defer clus.Terminate(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
I follow the logic from 
```
func TestV3PutOnNonExistLease(t *testing.T) {
	defer testutil.AfterTest(t)
	clus := NewClusterV3(t, &ClusterConfig{Size: 1})
	defer clus.Terminate(t)

	ctx, cancel := context.WithCancel(context.Background())
	defer cancel()

	badLeaseID := int64(0x12345678)
	putr := &pb.PutRequest{Key: []byte("foo"), Value: []byte("bar"), Lease: badLeaseID}
	_, err := toGRPC(clus.RandClient()).KV.Put(ctx, putr)
	if !eqErrGRPC(err, rpctypes.ErrGRPCLeaseNotFound) {
		t.Errorf("err = %v, want %v", err, rpctypes.ErrGRPCLeaseNotFound)
	}
}
```

Since it has the context with cancel so I used it as well. I Just want to make sure the code is consistent with other test cases